src/fheroes2/game/game_startgame.cpp

```diff
@@ -789,8 +794,23 @@ fheroes2::GameMode Interface::Basic::HumanTurn( bool isload )
         // hotkeys
         if ( le.KeyPress() ) {
             // if the hero is currently moving, pressing any key should stop him
-            if ( isMovingHero )
+            if ( isMovingHero ) {
                 stopHero = true;
+            }
+
+#if defined( WITH_DEBUG )
+            else if ( HotKeyPressEvent( Game::HotKeyEvent::TRANSFER_CONTROL_TO_AI ) ) {
+                if ( fheroes2::showMessage( fheroes2::Text( _( "Warning" ), fheroes2::FontType::normalYellow() ),
+                                            fheroes2::Text( _( "Do you want to transfer control from you to the AI? The effect will take place only on the next turn." ),
```

Comment:
Related to the above comment: the AI takes control immediately, not the next turn.
```suggestion
                                            fheroes2::Text( _( "Do you want to transfer control from you to the AI?" ),
```

Reply:
It is on the next turn. Please check this video:

https://user-images.githubusercontent.com/19829520/192236982-848c8148-005a-493c-a3c2-ec748f75c78c.mp4

Reply:
Ah you're right. My bad then. It looked like it happened on the same day for me.

Reply:
Sure. I will add an extra sentence.